card/mod/05_standard/set/type/set.rb

```diff
@@ -47,17 +47,19 @@
       %{
         <div class="template-editor-left">{{</div> 
         <div class="template-editor-main">
-          #{
-            frame :no_slot=>true, :title=>card.label, :menu_hack=>:template_closer do
-              _render_core args.merge(:unlabeled=>true)
-            end
-          }
+          #{ render_template_editor_frame args }
         </div>
         <div class="template-editor-right">}}</div> 
       }
     end
   end
   
+  view :template_editor_frame do |args|
```

Comment:
this fixes some css issues, because when this was in the template_editor view, the slot got the "template_editor-view" class, too.  caused much weirdness.